.travis.yml

```diff
@@ -47,8 +49,7 @@ install:
     - export LINKFLAGS="${LDFLAGS}"
     - mkdir build
     - cd build
-    - cmake -DDOWNLOAD_GTEST=ON -DCMAKE_PREFIX_PATH=$CONDA_PREFIX ..;
+    - cmake -DBUILD_TESTS=ON ..
```

Comment:
Why changing this?

Reply:
I just reverted this change, I changed it because otherwise it gives [this error](https://travis-ci.com/github/xtensor-stack/xtensor-io/jobs/372634207):
```
In file included from /home/travis/build/xtensor-stack/xtensor-io/build/test/googletest-src/googletest/src/gtest-all.cc:41:
/home/travis/build/xtensor-stack/xtensor-io/build/test/googletest-src/googletest/src/gtest.cc:2112:72: error: conversion to 'long unsigned int' from 'int' may change the sign of the result [-Werror=sign-conversion]
 2112 | std::vector<std::string> ArrayAsVector(const char* const (&array)[kSize]) {
      |                  
```
It looks like you fixed this bug in https://github.com/google/googletest/pull/2722, but it has not been released yet?

Reply:
You can use the branch google:v1.10.x or the warning branch from my fork.